client/src/components/inventory/HardwareLocationContents.tsx

```diff
@@ -61,7 +61,7 @@ const HardwareLocationContents = ({
             {filteredItems(itemByCat.items, searchQuery).length ? (
               <>
                 <Accordion.Title
-                  key={`${itemsByLocation.location.location_id}-${index}-title`}
+                  key={`${itemsByLocation.location.location_id}-title`}
```

Comment:
This and line 77 of this file might be problematic if their keys are not unique

Reply:
So for React, you're not supposed to use the index as part of the key, because the layout of the items can change and they will be the same, but the index will change and it will cause an unnecessary rerender. As such, you're not supposed to include the index as part of the key.
<img width="922" alt="Screen Shot 2021-04-16 at 6 03 39 PM" src="https://user-images.githubusercontent.com/8273038/115088396-13f9d300-9ede-11eb-9dcd-c1d828b53879.png">

Reply:
Yeah, the keys were like that because the location ID alone in the accordian was not unique.  it probably isn't hurting anything although we should fix it eventually

Reply:
Oh ok yeah, I can go through and fix all the keys and rendering stuff after this PR bc there are some other stuff I noticed too. Though this stuff isn't really that important at our level anyways